one-pager-maker/src/pages/Edit.tsx

```diff
@@ -63,6 +67,18 @@ function Edit() {
     }
   }
 
+  if (documentResult.data?.error && documentResult.data?.error.code === 'permission-denied') {
+    return (
+        <main className={'w-screen pt-48 flex flex-col justify-center gap-12 bg-background'}>
+          <h1 className={'text-4xl text-center font-bold'}>403 Forbidden</h1>
+          <p className={'text-2xl text-center'}>
```

Comment:
Although it is not limited to here, Why do you surround it with {} when the content is just tailwind?

Reply:
It's just because my IDE does auto complete like this, but it seems like we shouldn't use {} for string literal.

https://qiita.com/baby-degu/items/ea4eede60bbe9c63a348#9-%E6%96%87%E5%AD%97%E5%88%97%E5%B1%9E%E6%80%A7%E3%81%AB%E6%B3%A2%E6%8B%AC%E5%BC%A7%E3%81%AF%E4%B8%8D%E8%A6%81

Reply:
Oh, IDE...